src/q7/matrix-INMA2380/exercises/ch1.tex

```diff
@@ -110,92 +201,92 @@ \section{Matrix' algebra}
     & = A^{p+q}\\
     & = \underbrace{A \cdots A}_{q+p}\\
     & = \underbrace{A \cdots A}_{q} \underbrace{A \cdots A}_{p}\\
-    & = A^qA^p\\
+    & = A^qA^p.\\
     (A^p)^q
     & = \underbrace{\underbrace{A \cdots A}_{p} \cdots \underbrace{A \cdots A}_{p}}_q\\
     & = \underbrace{A \cdots A}_{pq}\\
-    & = A^{pq}
+    & = A^{pq}.
   \end{align*}
 \end{solution}
 
 \exo{2}
-Let's consider (upper) triangular Toeplitz,
-i.e. the (upper) triangular matrices having equal elements
- along the diagonal ($t_{ij} = t_{i+k,j+k}$):
+Consider the set of upper triangular Toeplitz matrices, i.e., the upper triangular matrices with equal elements
+ along the diagonals ($t_{ij} = t_{i+k,j+k}$):
 \[
   T =
-  \begin{pmatrix}
+  \begin{bmatrix}
     t_1 & t_2 & \cdots & t_n\\
     0   & t_1 & \ddots & \vdots\\
     \vdots & \ddots & \ddots & t_2\\
     0 & \cdots & 0 & t_1
-  \end{pmatrix}.
+  \end{bmatrix}.
 \]
-Prove that these matrices commute.
+Show that these matrices commute.
 
 \begin{solution}
-  Let $T,U$ be upper triangular Toeplitz,
+  Let $T,U$ be upper triangular Toeplitz matrices.
+  Then
```

Comment:
If you don't want to put such a large `\vfuzz` value, you can try creating 2 paragraphs here, so that the solution header appears on PDF page 8 (well, 7 not counting the front page). But it looks kinda ugly ^^

Reply:
Correction: you can then even remove entirely the `vfuzz=`. But I still find it ugly.